src/components/Dialogs/FileBrowser/FileBrowserDialogComponent.tsx

```diff
@@ -195,7 +195,7 @@ export class FileBrowserDialogComponent extends React.Component<{ appStore: AppS
             lazy: true,
             isOpen: fileBrowserStore.fileBrowserDialogVisible,
             onClose: fileBrowserStore.hideFileBrowser,
-            onOpened: () => fileBrowserStore.getFileList(fileBrowserStore.startingDirectory),
+            onOpened: () => {const _list = fileBrowserStore.fileList; fileBrowserStore.fileList = null; fileBrowserStore.fileList = _list; },
```

Comment:
this should be a private class function instead of an anonymous function. The frontend also doesn't use `_` for any varible names, so rather avoid that and use a descriptive name for it. Maybe `tempList`?

This all looks a bit hacky. `fileBrowserStore.fileList` is an observable, I'd rather we edited it throuh an `@action`

Reply:
Fixed it. I will looking into a better solution behind blueprint.